packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -42,7 +47,6 @@ interface WaferMapArgs {
     gridMaxY: number | undefined;
     dieHover: unknown;
     validity: WaferMapValidity;
```

Comment:
Reminder to document the new validity flag in storybook alongside the existing one.